options/locale/locale_en-US.ini

```diff
@@ -1294,6 +1296,8 @@ settings.protect_merge_whitelist_committers = Enable Merge Whitelist
 settings.protect_merge_whitelist_committers_desc = Allow only whitelisted users or teams to merge pull requests into this branch.
 settings.protect_merge_whitelist_users = Whitelisted users for merging:
 settings.protect_merge_whitelist_teams = Whitelisted teams for merging:
+settings.protect_check_status_contexts = Status checks found in the last week for this repository
+settings.protect_check_status_contexts_desc = Require status checks to pass before merging Choose which status checks must pass before branches can be merged into a branch that matches this rule. When enabled, commits must first be pushed to another branch, then merged or pushed directly to a branch that matches this rule after status checks have passed.
```

Comment:
```suggestion
settings.protect_check_status_contexts_desc = Choose which status checks must pass before a pull-request can be merged.
```

Reply:
The description comes from github originally.